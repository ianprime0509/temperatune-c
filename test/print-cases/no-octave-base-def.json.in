{
  "name": "Octave base not defined",
  "octaveBaseName": "C",
  "referencePitch": 440,
  "referenceName": "A",
  "referenceOctave": 4,
  "notes": {
    "A": ["A", 0]
  }
}
