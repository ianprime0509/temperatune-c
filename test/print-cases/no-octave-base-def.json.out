temperatune: could not determine offset of octave base 'C'
